### ISSUE TITLE
add Welcome Bot to Parsons Slack

### ISSUE THREAD

**Issue by shaunagm** [move-coop/parsons-community#12]
Sharine was working on this before she joined TMC but the custom bot she built was just not working for some reason. Should we abandon the custom attempt at find some other app?

**Comment by bmos** [comment]
What features are you looking for?
Just messaging new users with a welcome message?

**Comment by shaunagm** [comment]
Yes, essentially, and possibly also the ability to start DMs with a community member who can serve as a sort of mentor or welcomer.

We'd been trying to build a custom app and ran into permissions issues, but there's likely a premade app we can install

**Comment by shaunagm** [comment]
Thinking about this a bit further, we might want something more sophisticated (if available) that could help manage the whole onboarding workflow (see #31). For example, maybe when a person joins Slack they get an automatic welcome message asking them to fill out the onboarding form...or maybe there's a way for them to respond to questions from the bot and then those get stored externally? And then based on whether they have most interest in development, community, or learning, it would tag me, Sharine, or Kasia.
